docs/contributing/plugin_api/index.rst

```diff
@@ -1,6 +1,12 @@
 Plugin API
 ==========
 
+The Pulp Plugin API is versioned separately from Pulp Core. It is governed by `semantic
+versioning <http://semver.org/>`_. Backwards incompatible changes may be made while the Plugin
+API is in the alpha stage. The Plugin API will be considered stable when it reaches v1.0.
```

Comment:
Semantic versioning doesn't define an alpha stage. We define an alpha stage as a code maturity statement, but I think semver only has "stable"( >= 1.0) and "not yet stable" ( < 1.0).

I think the sentence `Backwards incompatible changes may be made while the Plugin API is in the alpha stage. The Plugin API will be considered stable when it reaches v1.0.` should be rewritten to something like:

`Backwards incompatible changes may be made until the Plugin API reaches stability with v1.0`.